internal/parser/gen/gen.go

```diff
@@ -11,10 +11,8 @@
 //   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 //   See the License for the specific language governing permissions and
 //   limitations under the License.
-
-/*
-This package controls the generation of all parser-related code.
-*/
+//
+// This package controls the generation of all parser-related code.
```

Comment:
I guess it was nice that these were two different comment groups.
One for copyright and one to be attached to package doc for godoc generation